### DISCUSSION TITLE
ModuleNotFoundError: No module named 'fpdf'

### DISCUSSION THREAD

**Discussion by 88arvin** [py-pdf/fpdf2#748] [General]
getting the same error after pip installation in vscode :-

ModuleNotFoundError: No module named 'fpdf'

**Comment by Lucas-C** [comment]
Hi and welcome @88arvin 😊

It's okay, but you really did not provide enough information for me to help you 😢

When people report bugs / issues, we ask them to provide some detailed information to troubleshoot and reproduce the problem:
> If an exception is raised, it is very important that you provide the **full error message**.
> Otherwise members of the fpdf2 community won't be able to help you with your problem.

> Please include some minimal Python code reproducing your issue
> If you don't know how to build a minimal reproducible example, please check this tutorial: https://stackoverflow.com/help/minimal-reproducible-example

> Please provide the following information:
> * **Operating System**: Windows, Mac OSX, Linux flavour...
> * **Python version**: you can get this information with `python --version`
> * **`fpdf2` version used**: if you installed it with `pip`, you can get this information in `pip freeze` output

_( quoted from: https://github.com/PyFPDF/fpdf2/blob/master/.github/ISSUE_TEMPLATE/bug_report.md )_

You mentioned using VSCode.
Could you please also indicate how you installed `fpdf2`?
If you used `pip`, what was the command?

People sometimes install [PyPDF2](https://pypi.org/project/PyPDF2/) instead of `fpdf2` 😅
It's an easy typo to make.

And finally, have you checked the previous issues open about `ModuleNotFoundError`?
https://github.com/PyFPDF/fpdf2/issues?q=ModuleNotFoundError
The problem may have already been answered in the past
